### DISCUSSION TITLE
ControlNet Hands !!!

### DISCUSSION THREAD

**Discussion by NonaBuhtig** [Mikubill/sd-webui-controlnet#1561] [General]
Hi guys !
I just found that there is a controlnet for hands, would be huge to have it in the webui extension 🥇 

[https://huggingface.co/MakiPan/controlnet-encoded-hands-130k](https://huggingface.co/MakiPan/controlnet-encoded-hands-130k)
[https://huggingface.co/spaces/vllab/controlnet-hands](https://huggingface.co/spaces/vllab/controlnet-hands)

**Comment by p0mad** [comment]
How have you used it @NonaBuhtig ?

is it possible to manually install and use this? How?

Thanks

**Comment by 2blackbar** [comment]
Its not , when you load it and provide their image, it wouldnt work
